tests/ops/api/v1/endpoints/test_privacy_request_endpoints.py

```diff
@@ -1947,7 +1948,7 @@ def test_approve_privacy_request(
         privacy_request.save(db=db)
 
         payload = {
-            JWE_PAYLOAD_SCOPES: user.client.scopes,
+            JWE_PAYLOAD_ROLES: user.client.roles,
```

Comment:
The `user` fixture now only has roles not scopes, so this token should likewise have roles